models/iaf_cond_alpha.nestml

```diff
@@ -31,7 +31,7 @@ Author: Schrader, Plesser
 
 SeeAlso: iaf_cond_exp, iaf_cond_alpha_mc
 */
-neuron iaf_cond_alpha:
+/*neuron iaf_cond_alpha:
```

Comment:
wouldn't it be better to separate the models into two files and keeping them both available?